ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables-mpls-vpn.md

```diff
@@ -0,0 +1,130 @@
+# Fabric Variables
+
+- The fabric underlay and overlay topology variables, define the elements related to build the L3 Leaf and Spine fabric.
+- The following underlay routing protocols are supported:
+  - ISIS-SR.
+  - ISIS + LDP.
+  - ISIS-SR + LDP.
+  - OSPF + LDP.
+- The following overlay routing protocols are supported:
+  - IBGP (default)
+- Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
+  - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
+- The variables should be applied to all devices in the fabric.
+
+**Variables and Options:**
+
+```yaml
+# Underlay routing protocol | Required.
+underlay_routing_protocol: < ospf or isis | Default -> isis >
```

Comment:
Please think about using this variable for setting all the different variants. Ex. `isis-sr`, `isis-ldp` etc.

Reply:
Implemented this suggestion and removed booleans for isis_sr and mpls_ldp. Will update documentation accordingly.